website/docs/guides/best-practices/how-we-style/3-how-we-style-our-python.md

```diff
@@ -0,0 +1,39 @@
+---
+title: How we style our python
```

Comment:
Is this lowercase `python` a specific choice? everywhere else it's capitalised but as the lowercase-your-proper-nouns company, I'm loath to immediately label this a typo

Reply:
good q -- @matthewshaver what's our official style stance here do we capitalize python?